test_scripts/LowVoltage/002_Low_Voltage_and_Ignition_Off.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) SDL is started (there was no LOW_VOLTAGE signal sent)
+-- 2) There are following app’s in HMI levels:
+-- App1 is in FULL
+-- App2 is in LIMITED
+-- App3 is in BACKGROUND
+-- App4 is in NONE
+-- 3) All apps have some data that can be resumed
+-- 4) SDL get LOW_VOLTAGE signal
+-- 5) And then SDL get IGNITION_OFF signal
+-- 6) And then SDL is started
+-- 7) All apps are registered with the same hashID
```

Comment:
same question regarding `All apps are registered with the same hashID`

Reply:
@yang1070 Please see above